src/arrays.js

```diff
@@ -1,61 +1,73 @@
 const getNthElement = (index, array) => {
-  // your code here
+  
+  if (index >= array.length){
+    return array[index-array.length];
+  }
+  return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+const csv = array.toString();
+return(csv);
 };
 
 const csvStringToArray = string => {
-  // your code here
+const array = string.split(',');
+return(array);
 };
```

Comment:
as before, instead of assigning to a variable and then returning variable, just return what you would have assigned to the variable, e.g.
```
const arrayToCSVString = array => {
  return array.toString();
};
```